ludwig/schema/decoders/base.py

```diff
@@ -11,7 +11,7 @@
 
 
 @DeveloperAPI
-@dataclass(repr=False)
+@dataclass(repr=False, order=True)
```

Comment:
Idea: define this dataclass in `ludwig/schema/utils`.

```python
from marshmallow_dataclass import dataclass

ludwig_schema_dataclass = dataclass(repr=False, order=True)
```

Then in all subsequent schema dataclass definitions, use: `@ludwig_schema_dataclass`

Reply:
+1

Reply:
This is a great idea! I'll create a linear ticket so I can do this in a followup PR!